.github/workflows/create-release.yml

```diff
@@ -0,0 +1,55 @@
+name: Create Release
+on:
+  release:
+    types: [created]
+jobs:
+  build:
+    name: Create Release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Setup Java
+        uses: actions/setup-java@v1
+        with:
+          java-version: 8
+
+      - name: Publish plugin
+        uses: eskatos/gradle-command-action@v1.2.0
+        env:
+          GRADLE_PUBLISH_KEY: ${{ secrets.GRADLE_PUBLISH_KEY }}
+          GRADLE_PUBLISH_SECRET: ${{ secrets.GRADLE_PUBLISH_SECRET }}
+        with:
+          arguments: publishPlugins -Pgradle.publish.key=${GRADLE_PUBLISH_KEY} -Pgradle.publish.secret=${GRADLE_PUBLISH_SECRET}
```

Comment:
Ideally, these would be passed as environment variables instead of as command line arguments. Passing them as environment variables can leak the data to `ps`.

https://docs.gradle.org/current/userguide/build_environment.html#sec:project_properties

You should be able to set the environment variable `GRADLE_PUBLISH_KEY` and `GRADLE_PUBLISH_SECRET` and then this won't need to be passed as arguments.

Reply:
Thanks for the reference. Modify environment variables name as
```
ORG_GRADLE_PROJECT_gradle.publish.key
ORG_GRADLE_PROJECT_gradle.publish.secret
```